Bottom: 75136d30a288bb2a55aebf5e2cc22728ae0d27b5
Top:    f8869e89f35392daa2e2885e3ee328cb584feb18
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 23:08:24 +0100

Refresh of hid-gtk-render-a-line-along-si

---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 1afd211..7b089e9 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -570,6 +570,8 @@ ghid_layer_selector_new (void)
  *  If the user_id passed already exists in the layer selector, that layer
  *  will have its data overwritten with the new stuff.
  *
+ * XXX: Add feature to mark adjacent layers which belong to the same group
+ *
  *  \param [in] ls            The selector to be acted on
  *  \param [in] user_id       An ID used to identify the layer; will be passed to selection/visibility callbacks
  *  \param [in] name          The name of the layer; will be used on selector and menus
